src/tests/end-to-end/tests/details-view/fast-pass-report.test.ts

```diff
@@ -0,0 +1,103 @@
+        await detailsViewPage.waitForSelector(detailsViewSelectors.automatedChecksResultSection);
+        await openTabStopsPage(detailsViewPage);
+        await addTabStopsFailure(detailsViewPage);
+        await detailsViewPage.waitForAndClickSelector(detailsViewSelectors.exportReportButton);
+        await detailsViewPage.waitForSelector(detailsViewSelectors.singleExportToHtmlButton);
+        reportDownload = await detailsViewPage.downloadExportReport(
+            detailsViewSelectors.singleExportToHtmlButton,
+            reportSaveAsFilePath,
+        );
+        reportPage = await browser.newHtmlReportPage(reportDownloadUrl);
+    });
+
+    afterAll(async () => {
+        // delete GUID file auto-generated by the download
+        await detailsViewPage.deleteDownloadedFile(reportDownload);
+        // delete renamed HTML used to open report for testing
+        await detailsViewPage.deleteFile(reportSaveAsFilePath);
+        await browser?.close();
+    });
+
+    it('should pass accessibility validation', async () => {
+        const results = await scanForAccessibilityIssues(reportPage, '*');
+        // should be expect(results).toHaveLength(0), but because of existing accessibility issue
+        // in the report, it's currently a snapshot. Once fixed we can remove the snapshot and check
+        // the length of the results array instead.
+        expect(results).toMatchSnapshot();
+    });
+
+    it('renders', async () => {
+        await reportPage.waitForSelector(fastPassReportSelectors.reportHeaderSection);
+    });
+});
+
+async function openTabStopsPage(detailsViewPage: DetailsViewPage): Promise<void> {
+    await detailsViewPage.waitForAndClickSelector(tabStopsSelectors.navDataAutomationId);
+}
+
+async function addTabStopsFailure(detailsViewPage: DetailsViewPage): Promise<void> {
+    const initialFailureInstanceText = 'this is a test failure instance';
+
+    //click "Fail" radio
+    await detailsViewPage.waitForAndClickSelector(tabStopsSelectors.tabStopsFailRadioButton);
+
+    //click "+" button
+    await detailsViewPage.waitForAndClickSelector(tabStopsSelectors.addFailureInstanceButton);
+
+    //add text to TextArea in failed instances panel
+    const addFailureTextArea = await detailsViewPage.waitForSelector(
+        tabStopsSelectors.addFailedInstanceTextArea,
+    );
+    await addFailureTextArea.fill(initialFailureInstanceText);
+
+    //click add button
+    await detailsViewPage.clickSelector(tabStopsSelectors.primaryAddFailedInstanceButton);
+
+    //check for failed instances section
+    await detailsViewPage.waitForSelector(tabStopsSelectors.failedInstancesSection);
+
+    // expand collapsible content to reveal failed instance
+    await detailsViewPage.clickSelector(tabStopsSelectors.collapsibleComponentExpandToggleButton);
+
+    //await text of failed instance
+    await detailsViewPage.waitForSelector(tabStopsSelectors.instanceTableTextContent);
```

Comment:
It might be a good smoke test to include a check here that the actual text content matches `initialFailureInstanceText`

Reply:
This is handled now with the refactor of the tabstops e2e test. Part of the `addFailedTabStopsInstance` function is testing that the text content matches.